specification/_types/common.ts

```diff
@@ -315,3 +315,15 @@ export class IndicesOptions {
    */
   ignore_throttled?: boolean
 }
+
+/**
+ * Uses sliced scroll to parallelize process. Using `auto` chooses a reasonable number for most data streams
+ * and indices.
+ *
+ * @codegen_names value, auto
```

Comment:
Does the name `auto` here still make sense if another value is added to the enum?

Reply:
It does, `auto` is a special value for this field which is translated to `0` on ES side. Providing `0` in the payload doesn't apply the same behavior.

Reply:
After a discussion with @technige we ended up considering that `all` is the name of a function used to compute the number of slices. There's only one such function as of now, but there could be others. So this becomes: 

```ts
/**
 * Slices configuration used to parallelize a process.
 *
 * @codegen_names value, computed
 */
export type Slices = integer | SlicesCalculation

/**
 * Named computations to calculate the number of slices
 */
export enum SlicesCalculation {
  /**
   * Let Elasticsearch choose a reasonable number for most data streams and indices.
   */
  auto
}
```